root@f14be223f84e:/# pg_restore -U admin -h localhost --verbose --create --dbname=gis '/share/gis_2022-08-24.dump'
pg_restore: connecting to database for restore
pg_restore: creating DATABASE "gis"
pg_restore: while PROCESSING TOC:
pg_restore: from TOC entry 5770; 1262 16386 DATABASE gis admin
pg_restore: error: could not execute query: ERROR:  database "gis" already exists
Command was: CREATE DATABASE gis WITH TEMPLATE = template0 ENCODING = 'UTF8' LC_COLLATE = 'C.UTF-8' LC_CTYPE = 'C.UTF-8';


pg_restore: connecting to new database "gis"
pg_restore: creating DATABASE PROPERTIES "gis"
pg_restore: connecting to new database "gis"
pg_restore: creating EXTENSION "pg_cron"
pg_restore: from TOC entry 5; 3079 18269 EXTENSION pg_cron 
pg_restore: error: could not execute query: ERROR:  could not open extension control file "/usr/share/postgresql/14/extension/pg_cron.control": No such file or directory
Command was: CREATE EXTENSION IF NOT EXISTS pg_cron WITH SCHEMA public;


pg_restore: creating COMMENT "EXTENSION pg_cron"
pg_restore: from TOC entry 5772; 0 0 COMMENT EXTENSION pg_cron 
pg_restore: error: could not execute query: ERROR:  extension "pg_cron" does not exist
Command was: COMMENT ON EXTENSION pg_cron IS 'Job scheduler for PostgreSQL';


pg_restore: creating SCHEMA "topology"
pg_restore: from TOC entry 12; 2615 18088 SCHEMA topology postgres
pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER SCHEMA topology OWNER TO postgres;

pg_restore: creating COMMENT "SCHEMA topology"
pg_restore: creating EXTENSION "hstore"
pg_restore: creating COMMENT "EXTENSION hstore"
pg_restore: creating EXTENSION "postgis"
pg_restore: creating COMMENT "EXTENSION postgis"
pg_restore: creating EXTENSION "postgis_topology"
pg_restore: creating COMMENT "EXTENSION postgis_topology"
pg_restore: creating FUNCTION "public.asbinary(public.geometry)"
pg_restore: from TOC entry 1622; 1255 18232 FUNCTION asbinary(public.geometry) postgres
pg_restore: error: could not execute query: ERROR:  could not access file "$libdir/postgis-2.5": No such file or directory
Command was: CREATE FUNCTION public.asbinary(public.geometry) RETURNS bytea
    LANGUAGE c IMMUTABLE STRICT
    AS '$libdir/postgis-2.5', 'LWGEOM_asBinary';


pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER FUNCTION public.asbinary(public.geometry) OWNER TO postgres;

pg_restore: creating FUNCTION "public.asbinary(public.geometry, text)"
pg_restore: from TOC entry 1623; 1255 18233 FUNCTION asbinary(public.geometry, text) postgres
pg_restore: error: could not execute query: ERROR:  could not access file "$libdir/postgis-2.5": No such file or directory
Command was: CREATE FUNCTION public.asbinary(public.geometry, text) RETURNS bytea
    LANGUAGE c IMMUTABLE STRICT
    AS '$libdir/postgis-2.5', 'LWGEOM_asBinary';


pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER FUNCTION public.asbinary(public.geometry, text) OWNER TO postgres;

pg_restore: creating FUNCTION "public.astext(public.geometry)"
pg_restore: from TOC entry 1624; 1255 18234 FUNCTION astext(public.geometry) postgres
pg_restore: error: could not execute query: ERROR:  could not access file "$libdir/postgis-2.5": No such file or directory
Command was: CREATE FUNCTION public.astext(public.geometry) RETURNS text
    LANGUAGE c IMMUTABLE STRICT
    AS '$libdir/postgis-2.5', 'LWGEOM_asText';


pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER FUNCTION public.astext(public.geometry) OWNER TO postgres;

pg_restore: creating FUNCTION "public.estimated_extent(text, text)"
pg_restore: from TOC entry 1626; 1255 18236 FUNCTION estimated_extent(text, text) postgres
pg_restore: error: could not execute query: ERROR:  could not access file "$libdir/postgis-2.5": No such file or directory
Command was: CREATE FUNCTION public.estimated_extent(text, text) RETURNS public.box2d
    LANGUAGE c IMMUTABLE STRICT SECURITY DEFINER
    AS '$libdir/postgis-2.5', 'geometry_estimated_extent';


pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER FUNCTION public.estimated_extent(text, text) OWNER TO postgres;

pg_restore: creating FUNCTION "public.estimated_extent(text, text, text)"
pg_restore: from TOC entry 1625; 1255 18235 FUNCTION estimated_extent(text, text, text) postgres
pg_restore: error: could not execute query: ERROR:  could not access file "$libdir/postgis-2.5": No such file or directory
Command was: CREATE FUNCTION public.estimated_extent(text, text, text) RETURNS public.box2d
    LANGUAGE c IMMUTABLE STRICT SECURITY DEFINER
    AS '$libdir/postgis-2.5', 'geometry_estimated_extent';


pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER FUNCTION public.estimated_extent(text, text, text) OWNER TO postgres;

pg_restore: creating FUNCTION "public.geomfromtext(text)"
pg_restore: from TOC entry 1628; 1255 18238 FUNCTION geomfromtext(text) postgres
pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER FUNCTION public.geomfromtext(text) OWNER TO postgres;

pg_restore: creating FUNCTION "public.geomfromtext(text, integer)"
pg_restore: from TOC entry 1627; 1255 18237 FUNCTION geomfromtext(text, integer) postgres
pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER FUNCTION public.geomfromtext(text, integer) OWNER TO postgres;

pg_restore: creating FUNCTION "public.hexagon(integer, integer, double precision)"
pg_restore: creating FUNCTION "public.hexagoncoordinates(public.geometry, double precision)"
pg_restore: creating FUNCTION "public.ndims(public.geometry)"
pg_restore: from TOC entry 1629; 1255 18239 FUNCTION ndims(public.geometry) postgres
pg_restore: error: could not execute query: ERROR:  could not access file "$libdir/postgis-2.5": No such file or directory
Command was: CREATE FUNCTION public.ndims(public.geometry) RETURNS smallint
    LANGUAGE c IMMUTABLE STRICT
    AS '$libdir/postgis-2.5', 'LWGEOM_ndims';


pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER FUNCTION public.ndims(public.geometry) OWNER TO postgres;

pg_restore: creating FUNCTION "public.rowjsonb_to_geojson(jsonb, text)"
pg_restore: creating FUNCTION "public.setsrid(public.geometry, integer)"
pg_restore: from TOC entry 1630; 1255 18240 FUNCTION setsrid(public.geometry, integer) postgres
pg_restore: error: could not execute query: ERROR:  could not access file "$libdir/postgis-2.5": No such file or directory
Command was: CREATE FUNCTION public.setsrid(public.geometry, integer) RETURNS public.geometry
    LANGUAGE c IMMUTABLE STRICT
    AS '$libdir/postgis-2.5', 'LWGEOM_set_srid';


pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER FUNCTION public.setsrid(public.geometry, integer) OWNER TO postgres;

pg_restore: creating FUNCTION "public.srid(public.geometry)"
pg_restore: from TOC entry 1631; 1255 18241 FUNCTION srid(public.geometry) postgres
pg_restore: error: could not execute query: ERROR:  could not access file "$libdir/postgis-2.5": No such file or directory
Command was: CREATE FUNCTION public.srid(public.geometry) RETURNS integer
    LANGUAGE c IMMUTABLE STRICT
    AS '$libdir/postgis-2.5', 'LWGEOM_get_srid';


pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER FUNCTION public.srid(public.geometry) OWNER TO postgres;

pg_restore: creating FUNCTION "public.st_asbinary(text)"
pg_restore: from TOC entry 1632; 1255 18242 FUNCTION st_asbinary(text) postgres
pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER FUNCTION public.st_asbinary(text) OWNER TO postgres;

pg_restore: creating FUNCTION "public.st_astext(bytea)"
pg_restore: from TOC entry 1549; 1255 18243 FUNCTION st_astext(bytea) postgres
pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER FUNCTION public.st_astext(bytea) OWNER TO postgres;

pg_restore: creating FUNCTION "public.tilews(integer, integer, integer, integer)"
pg_restore: creating OPERATOR FAMILY "public.gist_geometry_ops"
pg_restore: from TOC entry 3404; 2753 18244 OPERATOR FAMILY gist_geometry_ops postgres
pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER OPERATOR FAMILY public.gist_geometry_ops USING gist OWNER TO postgres;

pg_restore: creating OPERATOR CLASS "public.gist_geometry_ops"
pg_restore: from TOC entry 3270; 2616 18245 OPERATOR CLASS gist_geometry_ops postgres
pg_restore: error: could not execute query: ERROR:  role "postgres" does not exist
Command was: ALTER OPERATOR CLASS public.gist_geometry_ops USING gist OWNER TO postgres;

pg_restore: creating TABLE "public.operator_stats"
pg_restore: creating TABLE "public.operator_stats_annual"
pg_restore: creating TABLE "public.operator_stats_monthly"
pg_restore: creating TABLE "public.rgns_h3"
pg_restore: creating TABLE "public.rgns_h3_segsum"
pg_restore: creating TABLE "public.rgns_hex_res4"
pg_restore: creating TABLE "public.rgns_hex_res4_last24hours"
pg_restore: creating TABLE "public.rgns_hex_res4_last30days"
pg_restore: creating TABLE "public.rgns_hex_res4_last5days"
pg_restore: creating TABLE "public.rgns_hex_res5_last24hours"
pg_restore: creating TABLE "public.rgns_hex_res5_last30days"
pg_restore: creating TABLE "public.rgns_hex_res5_last5days"
pg_restore: creating TABLE "public.rgns_hex_res6_last24hours"
pg_restore: creating TABLE "public.rgns_hex_res6_last30days"
pg_restore: creating TABLE "public.rgns_hex_res6_last5days"
pg_restore: creating TABLE "public.rgns_hex_sum"
pg_restore: creating TABLE "public.segs"
pg_restore: creating TABLE "public.ship_stats_annual"
pg_restore: creating TABLE "public.ship_stats_monthly"
pg_restore: creating TABLE "public.zone_segs"
pg_restore: creating TABLE "public.zones"
pg_restore: processing data for table "cron.job"
pg_restore: from TOC entry 5547; 0 18273 TABLE DATA job postgres
pg_restore: error: could not execute query: ERROR:  schema "cron" does not exist
Command was: COPY cron.job (jobid, schedule, command, nodename, nodeport, database, username, active) FROM stdin;
pg_restore: processing data for table "public.operator_stats"
pg_restore: processing data for table "public.operator_stats_annual"
pg_restore: processing data for table "public.operator_stats_monthly"
pg_restore: processing data for table "public.rgns_h3"
pg_restore: processing data for table "public.rgns_h3_segsum"
pg_restore: processing data for table "public.rgns_hex_res4"
pg_restore: processing data for table "public.rgns_hex_res4_last24hours"
pg_restore: processing data for table "public.rgns_hex_res4_last30days"
pg_restore: processing data for table "public.rgns_hex_res4_last5days"
pg_restore: processing data for table "public.rgns_hex_res5_last24hours"
pg_restore: processing data for table "public.rgns_hex_res5_last30days"
pg_restore: processing data for table "public.rgns_hex_res5_last5days"
pg_restore: processing data for table "public.rgns_hex_res6_last24hours"
pg_restore: processing data for table "public.rgns_hex_res6_last30days"
pg_restore: processing data for table "public.rgns_hex_res6_last5days"
pg_restore: processing data for table "public.rgns_hex_sum"
pg_restore: processing data for table "public.segs"
pg_restore: processing data for table "public.ship_stats_annual"
pg_restore: processing data for table "public.ship_stats_monthly"
pg_restore: processing data for table "public.spatial_ref_sys"
pg_restore: processing data for table "public.zone_segs"
pg_restore: processing data for table "public.zones"
pg_restore: processing data for table "topology.topology"
pg_restore: processing data for table "topology.layer"
pg_restore: executing SEQUENCE SET jobid_seq
pg_restore: from TOC entry 5798; 0 0 SEQUENCE SET jobid_seq postgres
pg_restore: error: could not execute query: ERROR:  schema "cron" does not exist
LINE 1: SELECT pg_catalog.setval('cron.jobid_seq', 1, false);
                                 ^
Command was: SELECT pg_catalog.setval('cron.jobid_seq', 1, false);


pg_restore: creating INDEX "public.idx_operator_stats_annual_exclude_category"
pg_restore: creating INDEX "public.idx_operator_stats_annual_operator"
pg_restore: creating INDEX "public.idx_operator_stats_annual_vsr_region"
pg_restore: creating INDEX "public.idx_operator_stats_annual_year"
pg_restore: creating INDEX "public.idx_operator_stats_monthly_exclude_category"
pg_restore: creating INDEX "public.idx_operator_stats_monthly_month"
pg_restore: creating INDEX "public.idx_operator_stats_monthly_operator"
pg_restore: creating INDEX "public.idx_operator_stats_monthly_vsr_region"
pg_restore: creating INDEX "public.idx_operator_stats_monthly_year"
pg_restore: creating INDEX "public.idx_operator_stats_operator"
pg_restore: creating INDEX "public.idx_operator_stats_year"
pg_restore: creating INDEX "public.idx_rgns_h3_geom"
pg_restore: creating INDEX "public.idx_rgns_h3_h3id"
pg_restore: creating INDEX "public.idx_rgns_h3_h3res"
pg_restore: creating INDEX "public.idx_rgns_h3_rgn"
pg_restore: creating INDEX "public.idx_rgns_h3_segsum_h3id"
pg_restore: creating INDEX "public.idx_rgns_h3_segsum_h3res"
pg_restore: creating INDEX "public.idx_rgns_h3_segsum_rgn"
pg_restore: creating INDEX "public.idx_segs_date"
pg_restore: creating INDEX "public.idx_segs_geom"
pg_restore: creating INDEX "public.idx_segs_mmsi"
pg_restore: creating INDEX "public.idx_segs_region"
pg_restore: creating INDEX "public.idx_segs_seg_id"
pg_restore: creating INDEX "public.idx_segs_vsr_region"
pg_restore: creating INDEX "public.idx_ship_stats_annual_exclude_category"
pg_restore: creating INDEX "public.idx_ship_stats_annual_mmsi"
pg_restore: creating INDEX "public.idx_ship_stats_annual_operator"
pg_restore: creating INDEX "public.idx_ship_stats_annual_vsr_region"
pg_restore: creating INDEX "public.idx_ship_stats_annual_year"
pg_restore: creating INDEX "public.idx_ship_stats_monthly_exclude_category"
pg_restore: creating INDEX "public.idx_ship_stats_monthly_mmsi"
pg_restore: creating INDEX "public.idx_ship_stats_monthly_month"
pg_restore: creating INDEX "public.idx_ship_stats_monthly_operator"
pg_restore: creating INDEX "public.idx_ship_stats_monthly_vsr_region"
pg_restore: creating INDEX "public.idx_ship_stats_monthly_year"
pg_restore: creating INDEX "public.idx_zone_segs_geom"
pg_restore: creating INDEX "public.idx_zone_segs_mmsi"
pg_restore: creating INDEX "public.idx_zone_segs_rgn"
pg_restore: creating INDEX "public.idx_zone_segs_timestamp"
pg_restore: creating INDEX "public.idx_zone_segs_zone"
pg_restore: creating INDEX "public.idx_zones_geom"
pg_restore: creating INDEX "public.idx_zones_rgn"
pg_restore: creating INDEX "public.idx_zones_zone"
pg_restore: creating POLICY "cron.job cron_job_policy"
pg_restore: from TOC entry 5743; 3256 18286 POLICY job cron_job_policy postgres
pg_restore: error: could not execute query: ERROR:  schema "cron" does not exist
Command was: CREATE POLICY cron_job_policy ON cron.job USING ((username = (CURRENT_USER)::text));


pg_restore: creating ROW SECURITY "cron.job"
pg_restore: from TOC entry 5742; 0 18273 ROW SECURITY job postgres
pg_restore: error: could not execute query: ERROR:  schema "cron" does not exist
Command was: ALTER TABLE cron.job ENABLE ROW LEVEL SECURITY;

pg_restore: creating ACL "public.TABLE operator_stats"
pg_restore: from TOC entry 5777; 0 0 ACL TABLE operator_stats admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.operator_stats TO replicator;


pg_restore: creating ACL "public.TABLE operator_stats_annual"
pg_restore: from TOC entry 5778; 0 0 ACL TABLE operator_stats_annual admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.operator_stats_annual TO replicator;


pg_restore: creating ACL "public.TABLE operator_stats_monthly"
pg_restore: from TOC entry 5779; 0 0 ACL TABLE operator_stats_monthly admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.operator_stats_monthly TO replicator;


pg_restore: creating ACL "public.TABLE rgns_h3"
pg_restore: from TOC entry 5780; 0 0 ACL TABLE rgns_h3 admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_h3 TO replicator;


pg_restore: creating ACL "public.TABLE rgns_h3_segsum"
pg_restore: from TOC entry 5781; 0 0 ACL TABLE rgns_h3_segsum admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_h3_segsum TO replicator;


pg_restore: creating ACL "public.TABLE rgns_hex_res4"
pg_restore: from TOC entry 5782; 0 0 ACL TABLE rgns_hex_res4 admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_hex_res4 TO replicator;


pg_restore: creating ACL "public.TABLE rgns_hex_res4_last24hours"
pg_restore: from TOC entry 5783; 0 0 ACL TABLE rgns_hex_res4_last24hours admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_hex_res4_last24hours TO replicator;


pg_restore: creating ACL "public.TABLE rgns_hex_res4_last30days"
pg_restore: from TOC entry 5784; 0 0 ACL TABLE rgns_hex_res4_last30days admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_hex_res4_last30days TO replicator;


pg_restore: creating ACL "public.TABLE rgns_hex_res4_last5days"
pg_restore: from TOC entry 5785; 0 0 ACL TABLE rgns_hex_res4_last5days admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_hex_res4_last5days TO replicator;


pg_restore: creating ACL "public.TABLE rgns_hex_res5_last24hours"
pg_restore: from TOC entry 5786; 0 0 ACL TABLE rgns_hex_res5_last24hours admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_hex_res5_last24hours TO replicator;


pg_restore: creating ACL "public.TABLE rgns_hex_res5_last30days"
pg_restore: from TOC entry 5787; 0 0 ACL TABLE rgns_hex_res5_last30days admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_hex_res5_last30days TO replicator;


pg_restore: creating ACL "public.TABLE rgns_hex_res5_last5days"
pg_restore: from TOC entry 5788; 0 0 ACL TABLE rgns_hex_res5_last5days admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_hex_res5_last5days TO replicator;


pg_restore: creating ACL "public.TABLE rgns_hex_res6_last24hours"
pg_restore: from TOC entry 5789; 0 0 ACL TABLE rgns_hex_res6_last24hours admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_hex_res6_last24hours TO replicator;


pg_restore: creating ACL "public.TABLE rgns_hex_res6_last30days"
pg_restore: from TOC entry 5790; 0 0 ACL TABLE rgns_hex_res6_last30days admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_hex_res6_last30days TO replicator;


pg_restore: creating ACL "public.TABLE rgns_hex_res6_last5days"
pg_restore: from TOC entry 5791; 0 0 ACL TABLE rgns_hex_res6_last5days admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_hex_res6_last5days TO replicator;


pg_restore: creating ACL "public.TABLE rgns_hex_sum"
pg_restore: from TOC entry 5792; 0 0 ACL TABLE rgns_hex_sum admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.rgns_hex_sum TO replicator;


pg_restore: creating ACL "public.TABLE segs"
pg_restore: from TOC entry 5793; 0 0 ACL TABLE segs admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.segs TO replicator;


pg_restore: creating ACL "public.TABLE ship_stats_annual"
pg_restore: from TOC entry 5794; 0 0 ACL TABLE ship_stats_annual admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.ship_stats_annual TO replicator;


pg_restore: creating ACL "public.TABLE ship_stats_monthly"
pg_restore: from TOC entry 5795; 0 0 ACL TABLE ship_stats_monthly admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.ship_stats_monthly TO replicator;


pg_restore: creating ACL "public.TABLE zone_segs"
pg_restore: from TOC entry 5796; 0 0 ACL TABLE zone_segs admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.zone_segs TO replicator;


pg_restore: creating ACL "public.TABLE zones"
pg_restore: from TOC entry 5797; 0 0 ACL TABLE zones admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: GRANT SELECT ON TABLE public.zones TO replicator;


pg_restore: creating DEFAULT ACL "public.DEFAULT PRIVILEGES FOR TABLES"
pg_restore: from TOC entry 3443; 826 18268 DEFAULT ACL DEFAULT PRIVILEGES FOR TABLES admin
pg_restore: error: could not execute query: ERROR:  role "replicator" does not exist
Command was: ALTER DEFAULT PRIVILEGES FOR ROLE admin IN SCHEMA public REVOKE ALL ON TABLES  FROM admin;
ALTER DEFAULT PRIVILEGES FOR ROLE admin IN SCHEMA public GRANT SELECT ON TABLES  TO replicator;


pg_restore: warning: errors ignored on restore: 52